pkg/plugins/runtime/k8s/webhooks/injector/testdata/inject.sidecar-feature.40.golden.yaml

```diff
@@ -51,20 +50,8 @@ spec:
       readOnly: true
   initContainers:
   - args:
-    - --redirect-outbound-port
-    - "15001"
-    - --redirect-inbound=true
-    - --redirect-inbound-port
-    - "15006"
-    - --kuma-dp-user
-    - "5678"
-    - --exclude-inbound-ports
-    - "9000"
-    - --exclude-outbound-ports
-    - ""
+    - --exclude-inbound-ports=9000
```

Comment:
are all the values gone because it's now the default?

Reply:
yes